app/res/values/theme.xml

```diff
@@ -21,8 +21,6 @@
         <item name="colorPrimaryDark">@color/primary_dark</item>
         <item name="colorAccent">@color/accent</item>
 
-        <item name="colorControlHighlight">@color/ripple_material_light</item>
```

Comment:
By removing this you changed the navigation drawer selector to white. It has not disappeared it just matches the background. It could be moved to the new navigation drawer style.